README.md

```diff
@@ -1,9 +1,937 @@
+| renderSelectSlotIndicator | `func`                                       | `null`         | Function to override the way the selected slot indicator is rendered. Given `{ time, column }`                                                                                                        |
+| renderStepDetail          | `func`                                       | `() => null`   | Custom function to render [`StepDetails`](#stepDetails). Given `stepDetails`                                                                                                                          |
+| scrollToTime              | `'firstEvent'`, `String`, `moment` or `Date` | `'firstEvent'` | Either `'firstEvent'` or [Date](#date) type. Scrolls to either the first event or a particular moment in time                                                                                         |
+| selectedDate              | `Date`, `moment` or `String`                 | `new Date()`   | The currently selected date. See [Date](#date)                                                                                                                                                        |
+| selectMinutes             | `Number`                                     | `15`           | Must be one of `[5, 10, 15, 20, 30, 60]`. The minute interval in which selecting clicking the calendar should occur in                                                                                |
+| stepDetails               | `Array`                                      | `null`         | Array of [`StepDetails`](#stepDetails)                                                                                                                                                                |
+| stepHeight                | `Number`                                     | `null`         | The height of each row in grid                                                                                                                                                                        |
+| stepMinutes               | `Number`                                     | `15`           | Must be one of `[5, 10, 15, 20, 30, 60]`. The minute interval between each horizontal line in the column                                                                                              |
+| visibleEventGroups        | `Number[]`                                   | `null`         | Determines which provided columns are rendered or hidden                                                                                                                                              |
+| withColumns               | `Bool`                                       | `true`         | TODO: Ask Brian                                                                                                                                                                                       |
```

Comment:
@bjenkins24 I'll need your help investigating exactly what this prop does - it's not obvious from example stories either